framework/src/modules/http_api/index.js

```diff
@@ -41,6 +41,7 @@ class HttpAPIModule extends BaseModule {
 			author: 'LiskHQ',
 			version: '0.1.0',
 			name: 'lisk-framework-http-api',
+			module: 'http_api',
```

Comment:
Its not required to add redundant information. You can access same with module `alias`.

```suggestion
```